03.CSS_class_manipulation/src/cssClassManipulationTs.ts

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const cssClass = {
+  add(
```

Comment:
no need to write every function with this newlines - it's for bigger ones only :D

Reply:
Wanted to make it more classy